config/initializers/ruby_stackoverflow.rb

```diff
@@ -0,0 +1,4 @@
+RubyStackoverflow.configure do|config|
+  config.client_key = ENV['CLIENT_KEY']
+  config.access_token = ENV['ACCESS_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.